hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/test/TestBatchSqlConnector.java

```diff
@@ -104,13 +105,14 @@ public static void create(
         String typesStringified = types.stream().map(type -> type.getTypeFamily().name()).collect(joining(DELIMITER));
         String valuesStringified = values.stream().map(row -> join(DELIMITER, row)).collect(joining(VALUES_DELIMITER));
 
-        sqlService.execute("CREATE MAPPING " + tableName + " TYPE " + TYPE_NAME
+        String sql = "CREATE MAPPING " + tableName + " TYPE " + TYPE_NAME
                 + " OPTIONS ("
                 + '\'' + OPTION_NAMES + "'='" + namesStringified + "'"
                 + ", '" + OPTION_TYPES + "'='" + typesStringified + "'"
                 + ", '" + OPTION_VALUES + "'='" + valuesStringified + "'"
-                + ")"
-        );
+                + ")";
+        System.out.println(sql);
```

Comment:
Deliberately left here?

Reply:
Yes. It helps when analyzing the tests, they often construct the query in a complex way so you can easily see what's actually executed.